misc/swagger-ballerina/modules/swagger-to-ballerina-generator/src/main/java/org/ballerinalang/swagger/cmd/SwaggerCmd.java

```diff
@@ -74,7 +73,7 @@
     @Override
     public void execute() {
         if (helpFlag) {
-            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(parentCmdParser, "swagger");
+            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo("swagger");
```

Comment:
Shall we move "swagger" to a constant? I think there are multiple place where we are using it..